src/components/ExchangeIssuance/ExchangeIssuanceWidget.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import IssueRedeemSelector from './components/IssueRedeemSelector'
+import TokenInputs from './components/TokenInputs'
+import OrderSummary from './components/OrderSummary'
+import IssueRedeemButton from './components/IssueRedeemButton'
+import HowToBuyLink from './components/HowToBuyLink'
+
+const ExchangeIssuanceWidget: React.FC = () => {
+  return (
+    <StyledBuySellCard>
+      <StyledBuySellCardContent>
+        <IssueRedeemSelector />
+        <TokenInputs />
+        <OrderSummary />
+        <IssueRedeemButton />
+        <HowToBuyLink />
+      </StyledBuySellCardContent>
+    </StyledBuySellCard>
+  )
+}
```

Comment:
Exchange Issuance Widget isn't rendering the `ExchangeSelector` component, so user's can't switch back to Uniswap trade

Reply:
fixed by wrapping exchange Selector outside buysell widget